engine/schema/src/main/java/com/cloud/vm/dao/ConsoleSessionDaoImpl.java

```diff
@@ -60,9 +65,20 @@ public int expungeSessionsOlderThanDate(Date date) {
 
     @Override
     public void acquireSession(String sessionUuid) {
+        TransactionLegacy txn = TransactionLegacy.currentTxn();
         ConsoleSessionVO consoleSessionVO = findByUuid(sessionUuid);
-        consoleSessionVO.setAcquired(new Date());
-        update(consoleSessionVO.getId(), consoleSessionVO);
+        long consoleSessionId = consoleSessionVO.getId();
+        try {
+            txn.start();
+            PreparedStatement preparedStatement = txn.prepareAutoCloseStatement(ACQUIRE_CONSOLE_SESSION);
+            preparedStatement.setDate(1, new java.sql.Date(-1L));
+            preparedStatement.setLong(2, consoleSessionId);
+            preparedStatement.executeUpdate();
+            txn.commit();
+        } catch (Exception e) {
+            txn.rollback();
+            LOGGER.warn(String.format("Failed acquiring console session id = %s: %s", consoleSessionId, e.getMessage()), e);
+        }
```

Comment:
the logic of this is the same as before. If this fixes the problem fine, but shouldn't we fix the update method instead, @nvazquez ?